src/GovUk.Education.ExploreEducationStatistics.Data.Model/Filter.cs

```diff
@@ -10,6 +10,7 @@ public class Filter
         public string? Hint { get; set; } 
         public string Label { get; set; } = string.Empty;
         public string Name { get; set; } = string.Empty;
+        public string? GroupCsvColumn { get; set; }
```

Comment:
Just wondering if this can be made non-nullable after [EES-4364](https://dfedigital.atlassian.net/browse/EES-4364) ?

Reply:
It will stay nullable because not all Filters have a grouping column.
Many filters don't have a grouping column, so all their filter items get grouped into in a default group labelled 'Default'.